doc/profiler.rst

```diff
@@ -51,6 +85,23 @@ Therefore, if ``print()`` called nothing else itself, its :guilabel:`Total Time`
 Meanwhile, :guilabel:`Total Time` for ``print_wrapper()`` would be 3 ms, but :guilabel:`Local Time` only 1 ms as the rest of that time was spend inside the ``print()`` function it called.
```

Comment:
I think the example in this paragraph should be shown in the image displayed above it; or it should be changed to describe what users are seeing in the image.

In my opinion, that would make the interpretation of results far more easier than it is right now.